internal/controllers/device_plugin_reconciler.go

```diff
@@ -136,16 +138,19 @@ type devicePluginReconcilerHelper struct {
 	client          client.Client
 	metricsAPI      metrics.Metrics
 	daemonSetHelper daemonSetCreator
+	nodeAPI         node.Node
 }
 
 func newDevicePluginReconcilerHelper(client client.Client,
 	metricsAPI metrics.Metrics,
-	scheme *runtime.Scheme) devicePluginReconcilerHelperAPI {
+	scheme *runtime.Scheme,
+	nodeAPI node.Node) devicePluginReconcilerHelperAPI {
```

Comment:
As @yevgeny-shnaidman mentioned, can we please make `scheme` the last input of this function?

Reply:
@ybettan done